### PR TITLE
Created initial wrappers for styling and applied basic bootstrap styles

### PR DESCRIPTION
**Pull request by talyh**
Components now wrap a "StyledComponent" version, which in turn renders a Bootstrap+Styled-Component implementation
(ie. Button wraps StyledButton which returns styled(React-Bootstrap.Button)

Bootstrap properties can be passed down (for instance Button` bsStyle="primary"` carries all the way down),
Conditional styling can be done in the StyledComponents based off props passed into it, in the form of `border: ${props => props['data-myprop'] ? '1px solid red' : '1px solid green'}`.
Additional handling of styling can be done in the StyledComponents, once we figure out layout guidelines.

**Changes since last time this PR was up**
- [x] Elements distribution in page - basic grid structure through Bootstrap+Styled-Component. Unlike components that would contain implementation and style (such as Button), Grid elements are solely stylistic, so no custom wrapper for StyledElement was created
- [x] map styling - not done with CSS, but rather through style property on the map object, so doesn't follow the Bootstrap+Styled-Component approach. See comments in code for more details.

**Pending**
- [ ] route buttons styling - **Dependency** - determination of links x onClick handlers for route changing

**WishList** 
- Use [Breadcrumbs](https://react-bootstrap.github.io/components/breadcrumb/) for routes

### CODE REVIEW

File: client/src/containers/MainPage.js [hunk 1]
```diff
@@ -39,10 +37,10 @@ class MainPage extends Component {
           inputValue={location}
           handleChange={this.handleChange}
         />
-        <Button><Link to="/report">Report Incident</Link></Button>
-        <Button><Link to="/data">View Reports</Link>
-
-        </Button>
+        {/* TH: inline style on Button >> Link only till we figure out whethher the routing will be done indeed through 
+          Links or via onClick handlers */}
+        <Button style="primary"><Link to="/report" style={{ color: 'white' }}>Report Incident</Link></Button>
```
Comment (heathercodes) [inline client/src/containers/MainPage.js]: I totally didn't catch this earlier, but it's an accessibility no-no to have links inside buttons, which this outputs into as HTML. Can we remove the button wrappings and maybe move the Button style to the Links if that's what's needed? Or else create a special "link looks like a button" class here to style these.

Ideally, the rule is Buttons should do things (e.g. onSubmit, increment counter) and Links should go somewhere (e.g. new window, new page)

Comment (talyh) [reply client/src/containers/MainPage.js]: @heathercodes  and @stephanie56 - not sure what was decided (if anything) about keeping or losing the links.
Theoretically I don't see an issue an issue with a button with an onClick handler determining route (assuming the router can handle this).
The wireframes and stories do mention buttons, so if we plan on replacing them with links, we should probably doublecheck with Purvi.

**Review by kaitlynmcgoldrick — COMMENTED** [review]
👍

**Review by kaitlynmcgoldrick — APPROVED** [review]
👍👍👍

**Pull request merged**